scrooge-generator/src/main/resources/javagen/service.java

```diff
@@ -91,3 +95,38 @@ public FinagledService (
   {{>finagleService}}
 {{/finagleServices}}
 }
+
+
+class TFCF<T> extends CompletableFuture<T> {
```

Comment:
Ideally we could move this out somewhere common so it isn't repeated over and over as an inner class